selfdrive/ui/qt/widgets/cameraview.cc

```diff
@@ -6,11 +6,7 @@
 namespace {
 
 const char frame_vertex_shader[] =
-#ifdef NANOVG_GL3_IMPLEMENTATION
```

Comment:
I think we still need this for mac, let's just give it a better name and add a comment to confirm that it's actually needed